staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -117,7 +120,7 @@ func testTryAcquireOrRenew(t *testing.T, objectType string) {
 				{
 					verb: "get",
 					reaction: func(action fakeclient.Action) (handled bool, ret runtime.Object, err error) {
-						return true, createLockObject(objectType, action.GetNamespace(), action.(fakeclient.GetAction).GetName(), rl.LeaderElectionRecord{}), nil
+						return true, createLockObject(objectType, action.GetNamespace(), action.(fakeclient.GetAction).GetName(), ojrl.LeaderElectionRecord{}), nil
```

Comment:
Why do we need to replace `rl` with `ojrl`, Is this a typo?

Reply:
sorry, it changed by my mistake.